third_party/terraform/resources/resource_iam_audit_config.go

```diff
@@ -39,35 +39,46 @@ var iamAuditConfigSchema = map[string]*schema.Schema{
 }
 
 func ResourceIamAuditConfig(parentSpecificSchema map[string]*schema.Schema, newUpdaterFunc newResourceIamUpdaterFunc, resourceIdParser resourceIdParserFunc) *schema.Resource {
+	return ResourceIamAuditConfigWithBatching(parentSpecificSchema, newUpdaterFunc, resourceIdParser, false /*enableBatching*/)
```

Comment:
One approach I've liked in cases like this is using an enum / some constants to name `true` something like `WithBatchingEnabled` and false `WithBatchingDisabled`.